### PR TITLE
Added Sonar configuration, ran spotless

### PR DESCRIPTION
**Pull request by Hilbrand**
Needs GitHub action to make it actually collect the Sonar results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # AERIUS Search
 
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=search&metric=alert_status)](https://sonarcloud.io/dashboard?id=search)
```
Comment (SerhatG) [inline README.md:3]:
```suggestion
[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=aerius_search&metric=alert_status)](https://sonarcloud.io/dashboard?id=aerius_search)
```

File: pom.xml [hunk 2]
```diff
@@ -68,6 +68,13 @@
 
     <aerius-tools.version>1.1.1</aerius-tools.version>
     <spotless.version>2.5.0</spotless.version>
+
+    <sonar.language>java</sonar.language>
+    <sonar.maven.plugin.version>3.7.0.1746</sonar.maven.plugin.version>
```
Comment (Gijsvanhorn) [inline pom.xml:73]: Can't we use the latest version of the plugin just like in the AERIUS-II repository?

**Review by Gijsvanhorn — APPROVED** [review]
LGTM

**Review by SerhatG — APPROVED** [review]
I'll add the GitHub action after the merge.

**Pull request merged**